src/ucp/rndv/rndv_mtype.inl

```diff
@@ -34,7 +34,8 @@ ucp_proto_rndv_mtype_init(const ucp_proto_init_params_t *init_params,
 
     if ((init_params->select_param->dt_class != UCP_DATATYPE_CONTIG) ||
         (ucp_proto_rndv_mtype_ep(worker, frag_mem_type, mem_type) == NULL) ||
-        !ucp_proto_init_check_op(init_params, UCP_PROTO_RNDV_OP_ID_MASK)) {
+        !ucp_proto_init_check_op_without_bounce(init_params,
```

Comment:
can we detect from init_params whether the protocol requires bounce buffer, and put this logic in the common function ucp_proto_init_check_op?